records_mover/records/job/hints.py

```diff
@@ -1,8 +1,15 @@
 """Defines hints supported by the job config parser."""
+from records_mover.records.delimited.hints import Hints
 from ...utils.json_schema import JsonParameter, JsonSchemaDocument
 from typing import Optional
 
 
+SUPPORTED_HINT_NAMES = [
+    'field-delimiter', 'compression', 'escape', 'quoting', 'encoding', 'header-row'
+]
+
+
+# TODO: This class isn't needed anymore if we can just provide Hint class
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker